src/components/EggheadPlayer/machine.ts

```diff
@@ -0,0 +1,63 @@
+import {Machine} from 'xstate'
```

Comment:
xstate and typescript requires some love: https://xstate.js.org/docs/guides/typescript.html